### PR TITLE
[Fixes 1249] Change Edit Layer style to Edit style

### PR DESCRIPTION
**Pull request by DavidQuartz**
'Edit Layer Style' has been changed to 'Edit Style' in ActionNav bar in resource view pages.
A new translation has been added for 'Edit Layer Style' for visual style editor plugin

### CODE REVIEW

File: geonode_mapstore_client/client/static/mapstore/translations/data.en-US.json [hunk 1]
```diff
@@ -144,7 +144,8 @@
             "export": "Export",
             "editInfo": "Edit Info",
             "editMetadata": "Edit Metadata",
-            "editStyle": "Edit layer style",
+            "editStyle": "Edit style",
+            "editLayerStyle": "Edit Layer style",
```
Comment (giohappy) [inline geonode_mapstore_client/client/static/mapstore/translations/data.en-US.json:148]: Edit Dataset style

File: geonode_mapstore_client/client/static/mapstore/translations/data.it-IT.json [hunk 2]
```diff
@@ -149,7 +149,8 @@
             "viewMetadata": "Visualizza i metadati",
             "viewData": "Visualizza i dati",
             "editInfo": "Modifica informazioni",
-            "editStyle": "Modifica stile di livello",
+            "editStyle": "Modifica Stile",
+            "editLayerStyle": "Modifica stile di livello",
```
Comment (giohappy) [inline geonode_mapstore_client/client/static/mapstore/translations/data.it-IT.json:153]: Modifica lo stile del dataset

**Review by giohappy — CHANGES_REQUESTED** [review]

**Review by giohappy — APPROVED** [review]

**Comment by DavidQuartz** [comment]
@marthamareal please test on master and let us know if we can backport on 4.0.x, thanks

**Comment by giohappy** [comment]
@DavidQuartz I just tested this and it looks ok, except "style" is lowercase, while the other menu items have the second term uppercase. I would fix this before merging to 4.0.x.

image

**Pull request merged**